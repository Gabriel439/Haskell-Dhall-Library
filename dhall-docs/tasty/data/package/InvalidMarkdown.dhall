-- Represents an status in our messaging system `
-- look, another line

let Pair = ./Pair.dhall

let DeliveryStatus =
    < Sent: Pair Natural Text {-| Message has been _sent_. It carries a ./Pair.dhall reference
                                    containing on the `first` field number of retries
                                    and on the `second` field the delivered text
                                -}
    | InProgress                -- | Message is on the way!
    | InQueue                   -- | Delivery for this message **hasn't** started
    >

in DeliveryStatus
