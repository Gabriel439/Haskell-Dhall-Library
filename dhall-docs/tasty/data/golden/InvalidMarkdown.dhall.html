<!DOCTYPE HTML>
<html
  ><head
    ><title
    >/InvalidMarkdown.dhall</title
    ><link href="index.css" type="text/css" rel="stylesheet"
    /><link href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;500;600;700&amp;family=Lato:ital,wght@0,400;0,700;1,400&amp;display=swap"
            type="text/css" rel="stylesheet"
    /><script src="index.js" type="text/javascript"
    /><meta charset="UTF-8"/></head
  ><body
    ><div class="nav-bar"
      ><img src="dhall-icon.svg" class="dhall-icon"
      /><p class="package-title"
      >test-package</p
      ><div class="nav-bar-content-divider"
      /><a id="switch-light-dark-mode" class="nav-option"
      >Switch Light/Dark Mode</a></div
    ><div class="main-container"
      ><h2 class="doc-title"
        ><span class="crumb-divider"
        >/</span
        ><a href="index.html"
        >test-package</a
        ><span class="crumb-divider"
        >/</span
        ><span href="index.html" class="title-crumb"
        >InvalidMarkdown.dhall</span></h2
      ><a data-path="/InvalidMarkdown.dhall" class="copy-to-clipboard"
        ><i
          ><small
          >Copy path to clipboard</small></i></a
      ><br
      /><div class="doc-contents"
      >Represents an status in our messaging system `
-- look, another line</div
      ><h3
      >Source</h3
      ><div class="source-code"
        ><pre
          ><span class="dhall-keyword"
          >let</span
          > <span class="dhall-label"
          >Pair</span
          > <span class="dhall-syntax"
          >=</span
          > ./Pair.dhall<br
          /><br
          /><span class="dhall-keyword"
          >let</span
          > <span class="dhall-label"
          >DeliveryStatus</span
          > <span class="dhall-syntax"
          >=</span
          > <span class="dhall-syntax"
          >&lt;</span
          > <span class="dhall-label"
          >Sent</span
          > <span class="dhall-syntax"
          >:</span
          > <span class="dhall-label"
          >Pair</span
          > <span class="dhall-builtin"
          >Natural</span
          > <span class="dhall-builtin"
          >Text</span
          > <span class="dhall-syntax"
          >|</span
          > <span class="dhall-label"
          >InProgress</span
          > <span class="dhall-syntax"
          >|</span
          > <span class="dhall-label"
          >InQueue</span
          > <span class="dhall-syntax"
          >&gt;</span
          ><br
          /><br
          /><span class="dhall-keyword"
          >in</span
          >  <span class="dhall-label"
          >DeliveryStatus</span></pre></div></div></body></html>